session/session.go

```diff
@@ -497,6 +498,7 @@ func (s *session) retry(ctx context.Context, maxCnt uint) error {
 			}
 			s.StmtCommit()
 		}
+		log.Warnf("con:%d txn(%d) has retried txn(%d)", connID, s.GetSessionVars().TxnCtx.StartTS, orgStartTS)
```

Comment:
s/`txn(%d)`/`txn:%d`?
I see other places use "txn:".

Reply:
@zimulala the other place is txn, this is txn start ts.
@lilin90 need help to translate like "transaction A which transaction start timestamp/transaction ID is xxx has finished (may be not success e.g. meets some errors) and it retried the statements which are from transaction B which transaction start timestamp/transaction ID is xxx"

Reply:
@jackysp I think it's easier to understand this way:

Transaction A, with the transaction start timestamp (transaction ID) xxx, has finished retrying (might fail, such as meeting some errors) all the statements in transaction B, with the transaction start timestamp (transaction ID) yyy.

Reply:
I've used the Unix naming method to keep it simple. PTAL @zimulala